compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.td

```diff
@@ -68,10 +68,10 @@ def DetachElementwiseFromNamedOps :
   let constructor = "mlir::iree_compiler::IREE::Flow::createDetachElementwiseFromNamedOpsPass()";
 }
 
-def DispatchLinalgOnTensors :
-    InterfacePass<"iree-flow-dispatch-linalg-on-tensors-pass", "mlir::FunctionOpInterface"> {
-  let summary = "Dispatch Linalg operations on tensors by using tile and distribute";
-  let constructor = "mlir::iree_compiler::IREE::Flow::createDispatchLinalgOnTensorsPass()";
+def FormDispatchRegions :
+    InterfacePass<"iree-flow-form-dispatch-regions-pass", "mlir::FunctionOpInterface"> {
```

Comment:
drop the "-pass" suffix on these (not sure why the original had them as we don't usually put -pass at the end)